docs/pages/versions/unversioned/sdk/sqlite.mdx

```diff
@@ -484,6 +495,19 @@ Check out the [Drizzle ORM documentation](https://orm.drizzle.team/) and the [`e
 
 Check out the [`expo-sqlite` integration guide](https://github.com/expo/knex-expo-sqlite-dialect) for more details.
 
+## SQLCipher
```

Comment:
![Screenshot 2025-04-01 at 17 42 30](https://github.com/user-attachments/assets/7b5382b8-cec5-4be6-b565-9f0704751b61)

Nit, looks like we need to add an expectation in the vale plugin config:
*https://github.com/expo/expo/blob/f8958ae3a7a0ecf8fa87ec08e69f085ec2a728f2/docs/.vale/writing-styles/expo-docs/HeadingCase.yml#L11-L11